marklogic-data-hub-central/ui/e2e/cypress/e2e/modeling/verifyConceptClassesInModeling.cy.tsx

```diff
@@ -365,4 +364,130 @@ describe("Concept classes in Modeling screen", () => {
     entityTypeTable.getConceptClass("TestConcept").should("not.exist");
   });
 
-});
+  it("Can delete concept class and relationship and publish without problems", () => {
+    cy.waitUntil(() => toolbar.getModelToolbarIcon()).click();
+    cy.wait(5000);
+    graphView.getAddButton().click();
+    graphView.getAddConceptClassOption().should("be.visible").click({force: true});
+    conceptClassModal.newConceptClassName("testDeleteStyle");
+    conceptClassModal.newConceptClassDescription("Created just to validate deleting.");
+    cy.waitUntil(() => conceptClassModal.getAddButton().click());
+    conceptClassModal.getAddButton().should("not.exist");
+
+    cy.wait(5000);
+    cy.log("**Create Concept Class**");
+    graphViewSidePanel.getSelectedConceptClassHeading("testDeleteStyle").should("exist");
+    graphViewSidePanel.getSelectedConceptClassHeadingInfo("testDeleteStyle").should("exist");
+    graphViewSidePanel.getConceptClassDescription().should("be.visible");
+    graphViewSidePanel.getConceptClassDescription().should("have.value", "Created just to validate deleting.");
+
+    modelPage.toggleColorSelector("testDeleteStyle");
```

Comment:
Again, I suggest we do this test one it above and re-use the Concept class alreadt created to optimize the times.
In this test we don't need to re-test things like picking colors and descriptions. We already covered this above.